pkg/ui/src/components/tooltip/tooltip.tsx

```diff
@@ -18,16 +18,18 @@ import "./tooltip.styl";
 export interface TooltipProps {
   children: React.ReactNode;
   theme?: "default" | "blue";
+  visible?: boolean;
```

Comment:
Our <Tooltip> component inherits props from AntTooltip (`TooltipProps & AntTooltipProps`) so it is already has `visible` and `defaultVisible` props. So it doesn't require making any changes here at all.